javascript-injectors/examples/cyfe-sign-in-via-credentials.js

```diff
@@ -0,0 +1,32 @@
+(function () {
+  const password = '<PASSWORD>'
+
+  function elExists (selector) {
+    const el = document.querySelector(selector)
+    return el !== undefined && el.getAttribute('aria-hidden') !== 'true'
+  }
+
+  function setValue (selector, value) {
+    const element = document.querySelector(selector)
+    element.value = value
+    element.dispatchEvent(new Event('change'))
+  }
+
+  function submitForm () {
+    document.querySelector('input[type="submit"]').click()
+  }
+
+  function login () {
+    try {
+      setValue('input[name="password"]', password)
+      submitForm()
+    } catch (error) {
+      console.warn(error)
+      setTimeout(login, 3000)
+    }
+  }
+
+  if (elExists('input[name=password]')) {
```

Comment:
Wrapping the login call inside elExists means login will be called only once depending on if the element exists or not.

Directly calling login() ensures that the retry logic is performed if the element appears later. 

So you might want to call login directly without checking if the element exists or not.